app/kumactl/cmd/install/render_files.go

```diff
@@ -44,7 +46,18 @@ type templateRenderer interface {
 }
 
 func simpleTemplateRenderer(text data.File) (templateRenderer, error) {
-	tmpl, err := template.New("").Funcs(sprig.TxtFuncMap()).Parse(string(text.Data))
+	tmpl, err := template.New("").Funcs(sprig.TxtFuncMap()).Funcs(template.FuncMap{
```

Comment:
This isn't necessary/shouldn't be here. The check should be only in `templates/_helpers.tpl`